scss/grid/_gutter.scss

```diff
@@ -6,19 +6,76 @@
 /// @group grid
 ////
 
+/// Get a gutter size for a given breakpoint
+/// @param {Keyword} $breakpoint [small] - Breakpoint name.
+/// @param {Number|Map} $gutters [$grid-column-gutter] - Gutter map or single value to use. Responsive gutter settings by default.
+///
+/// @returns {Number} Gutter size.
+@function grid-column-gutter(
+  $breakpoint: $-zf-zero-breakpoint,
+  $gutters: $grid-column-gutter
+) {
+  // If gutter is single value, return it
+  @if type-of($gutters) == 'number' {
+    @return $gutters;
+  }
+
+  // Else, return the corresponding responsive value
+  @return -zf-get-bp-val($gutters, $breakpoint);
+}
+
+/// Set the gutters on a column
+/// @param {Number|Keyword} $gutter [auto]
+///   Spacing between columns, accepts multiple values:
+///   - A single value will make the gutter that exact size.
+///   - A breakpoint name will make the gutter the corresponding size in the $gutters map.
+///   - "auto" will make the gutter responsive, using the $gutters map values.
+/// @param {Number|Map} $gutters [$grid-column-gutter] - Gutter map or single value to use. Responsive gutter settings by default.
+@mixin grid-column-gutter(
+  $gutter: auto,
+  $gutters: $grid-column-gutter
+) {
+  @if $gutter == auto and type-of($gutters) == 'map' {
+    // "auto"
+    @each $value, $breakpoint in $gutters {
+      @include breakpoint($breakpoint) {
+        @include grid-column-gutter($value);
+      }
+    }
+  } @else {
+    // breakpoint name
+    @if type-of($gutter) == 'string' {
+      $gutter: grid-column-gutter($gutter, $gutters);
```

Comment:
> With regards to the first, I've heard arguments for and against BEM - generally it feels unnecessarily verbose to me, though I'd be open to arguments why that verbosity is helpful.

BEM was not only created to help the developer knowing what he uses, but to prevent unwanted behaviours when using CSS classes : properties overwritten by specificity, naming collisions across components or frameworks... *I remember having these problems with Foundation*

Also, there is a lot of BEM grammars, some more verbose than others (i.e. `block-foo__element-bar--modifier-baz`, `BlockFoo_ElementBar-ModifierBaz`, etc...).